Web createapp use cookiecutter (#2396)
ome/omero-documentation: omero/developers/Web/Deployment.rst

From OMERO 5.6.0 release, the ``omero-web`` library supports Python 3 and

> can be installed via :command:`pip`. We need to specify a location :envvar:`OMERODIR`

Reviewer: ```suggestion
can be installed via :command:`pip`. You must specify an :envvar:`OMERODIR` location 
```

where `omero-web` will create log files and a :file:`config.xml` file. This can be any existing
directory.

We recommend you use a virtual environment and start by installing ``omero-py``

> as described at `omero-py <https://github.com/ome/omero-py>`_.

Reviewer: Pity that the README pointed to here is suggesting python 3.8 - will be EOL October this year.
